packages/components/organisms/f-checkout/src/components/tests/Checkout.test.js

```diff
@@ -32,6 +36,27 @@ describe('Checkout', () => {
         expect(wrapper.exists()).toBe(true);
     });
 
+    it('should show the login link.', () => {
+        const dummyName = 'Dave';
```

Comment:
I think there should already be a name that you can access in `defaultState.customer.firstName`